owlbot.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Google LLC
```

Comment:
Can we make this a git mv to preserve the license year, or update to the current year?

Reply:
Apparently git does not fully support this so we'll keep the year from the original synth.py.